cli/popper/commands/cmd_run.py

```diff
@@ -115,35 +115,29 @@
 def cli(ctx, **kwargs):
     """Executes one or more workflows and reports on their status.
 
-    [TARGET] : It can be either path to a workflow file or an action name.
-    If TARGET is a workflow, the workflow is executed.
-    If TARGET is an action, the specified action from the default workflow
-    will be executed.
+    ACTION : The action to execute from a workflow.
 
     Examples:
 
-    1. When no TARGET argument is passed, Popper will search for the
+    1. When no action argument is passed, Popper will search for the
```

Comment:
strictly speaking these are not examples but a description of what `popper run` does, so we could instead write these in narrative form instead of using a numbered list. Removing `Examples:` and then start a paragraph with

> When no `--wfile` is given, Popper will search for the... If an ACTION is passed, the specified action....

Also, since we don't do `--recursive`, the first sentence can be `Executes a github action workflow.`